ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml

```diff
@@ -49,6 +49,43 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
+- as: aws-ipi-disconnected-private-sno-basecap-none-arm-f28
```

Comment:
```suggestion
- as: aws-ipi-disc-sno-basecap-none-arm-f28
```

Reply:
thanks for the suggestion, changed the job name to `aws-ipi-disc-sno-baselinecaps-none-xxx-fxx`, no maximum length error any more in job config.